contracts/tests/conftest.py

```diff
@@ -72,7 +72,7 @@ def validators(accounts, malicious_validator_address):
     return accounts[:HONEST_VALIDATOR_COUNT] + [malicious_validator_address]
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture()
 def validator_set_contract_session(deploy_contract, validators, web3):
```

Comment:
```suggestion
def validator_set_contract(deploy_contract, validators, web3):
```

as I assume `session` refers to the scope

Reply:
This fixture is used in between every tests: https://github.com/trustlines-protocol/contract-deploy-tools/blob/master/deploy_tools/plugin.py#L107

So please use the scope `session` when deploying contracts so that the tests do not redeploy the contract each time. This will speed up testing.

Reply:
True. Solved. :)

Reply:
Oh. There is a reason because you should touch already approved code with care...

Reply:
@come I rebased the branch on itself and removed the change. Would love to see an approval, before I merge. Thanks for all the input. :v: